.travis.yml

```diff
@@ -12,6 +12,7 @@ install:
   - conda create --yes -n env_name python=$PYTHON_VERSION pip numpy scipy nose flake8
   - source activate env_name
   - pip install coveralls
+  - pip install https://github.com/biocore/burrito-fillings/archive/master.zip
```

Comment:
remove and include in setup.py, otherwise deblur is not going to be pip installable. See [here](https://github.com/biocore/qiime/blob/5719b22f782db204cbd1f4a2a68fda259d30cc5c/setup.py) to see an example on how to add a developer version in a setup.py